packages/core/strapi/lib/commands/routes/list.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const CLITable = require('cli-table3');
+const chalk = require('chalk');
+const { toUpper } = require('lodash/fp');
+
+const strapi = require('../../index');
+
+module.exports = async function() {
+  const app = await strapi().load();
+
+  const list = app.server.listRoutes();
+
+  const infoTable = new CLITable({
+    head: [chalk.blue('Method'), chalk.blue('Path')],
+    colWidths: [20, 80],
```

Comment:
`if colWidths is not specified, the layout manager will automatically widen rows to fit the content` ([doc](https://github.com/cli-table/cli-table3/blob/master/advanced-usage.md)).
What do you think about not putting widths to let it adapt?